backend/apid/graphql/handler_test.go

```diff
@@ -15,16 +16,21 @@ func TestHandlerTypeHandlersField(t *testing.T) {
 	handler := types.FixtureHandler("my-handler")
 	handler.Handlers = []string{"one", "two"}
 
-	client, factory := client.NewClientFactory()
-	impl := &handlerImpl{factory: factory}
+	client, _ := client.NewClientFactory()
+	impl := &handlerImpl{}
+
+	params := graphql.ResolveParams{}
+	params.Context = contextWithLoadersNoCache(context.TODO(), client)
```

Comment:
I'm not sure if this is that big of a deal, but I always deferred to using `context.Background()` in tests since the docs say that's one of the use cases for it https://godoc.org/golang.org/x/net/context#Background

Reply:
Good point, thanks for the callout. I'll change this.